exercises/practice/raindrops/raindrops.sql

```diff
@@ -0,0 +1,2 @@
+-- Schema: CREATE TABLE "raindrops" ("number" INT, "sound" TEXT);
+-- Task: update the raindrops table and set the sound based on the number.
```

Comment:
We should add short explanation to every student file. 
qhat do you think @BNAndras  @glennj